bandcamp_dl/__main__.py

```diff
@@ -84,6 +84,9 @@ def main():
         urls = Bandcamp.generate_album_url(arguments['--artist'], arguments['--track'], "track")
     elif arguments['--artist']:
         urls = Bandcamp.get_full_discography(arguments['--artist'], "music")
+        print(f"{len(urls)} albums/tracks have been found:")
```

Comment:
Other than this print I don't really have an issue with the PR. I just don't see a reason to clutter the terminal with anything more than the existing progress bar especially considering its not really a value add to print the urls.

If you want to dump them in debug that is fine though.

Reply:
Alright, I decided to remove it. It's already committed.